cmd/collector/cmd/example.go

```diff
@@ -36,15 +36,20 @@ var exampleCmd = &cobra.Command{
 
 		ctx := context.Background()
 
+		// Prepapre the logger.
+		zapLogger, _ := zap.NewProduction()
```

Comment:
I wonder if we should create a separate package for the logger that does the initialization. Similar to how Knative does it https://github.com/knative/pkg/blob/main/logging/logger.go

Reply:
Or just utilize it from an upstream project.

Reply:
Yeah, that looks nice. The Zap documentation is definitely not the best in the world and they do not mention anything about the best practices to initialize the logger. But if we find a good way to do so, I am happy to implement it.